From 0c49ba335d3ac00256fb8c70310c35482a0f1249 Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Tue, 2 May 2017 10:46:41 +0800
Subject: [PATCH] fiq_debugger: use __handle_sysrq instead of handle_sysrq

Because init.rc does the following operation, handle_sysrq
will do nothing. If we want to use sysrq, __handle_sysrq
can work.
write /proc/sys/kernel/sysrq 0

Change-Id: Ia51debd92f393326f183736e405e25dc4d6a2abc
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 drivers/staging/android/fiq_debugger/fiq_debugger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/android/fiq_debugger/fiq_debugger.c b/drivers/staging/android/fiq_debugger/fiq_debugger.c
index 7eb678b6b3eb..7615f2130ecd 100644
--- a/drivers/staging/android/fiq_debugger/fiq_debugger.c
+++ b/drivers/staging/android/fiq_debugger/fiq_debugger.c
@@ -373,7 +373,7 @@ static void fiq_debugger_do_sysrq(struct fiq_debugger_state *state, char rq)
 		return;
 	}
 	fiq_debugger_begin_syslog_dump(state);
-	handle_sysrq(rq);
+	__handle_sysrq(rq, false);
 	fiq_debugger_end_syslog_dump(state);
 }
 
-- 
2.35.3

